torch/_ops.py

```diff
@@ -1477,17 +1477,21 @@ def load_library(self, path):
 
         Args:
             path (str): A path to a shared library to load.
+
+        Returns:
+            ctypes.CDLL: The ctypes object for the loaded library.
         """
         path = _utils_internal.resolve_library_path(path)
         with dl_open_guard():
             # Import the shared library into the process, thus running its
             # static (global) initialization code in order to register custom
             # operators with the JIT.
             try:
-                ctypes.CDLL(path)
+                lib = ctypes.CDLL(path)
             except Exception as e:
                 raise OSError(f"Could not load this library: {path}") from e
         self.loaded_libraries.add(path)
+        return lib
```

Comment:
what do you plan to do with this?

Reply:
It's an internal use case, let me ping you in the workchat. But basically there are also extern "C" symbols that the program needs besides the torch custom ops. So `lib` would later used in `getattr(lib, "some_func_name")` to get the C function out in python.

Reply:
Why is calling CDLL again not a good solution here?

Reply:
Mainly want to avoid introducing the logic from L1484 to the CDLL callsite to parse the internal buck path. But open to suggestions.